client/src/components/Home/Home.js

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
 import IconSelector from './IconSelector';
 import SearchBar from './SearchBar';
 import './Home.css';
+import { Link } from 'react-router-dom';
+import BigRedButton from './../../images/bigRedButton.png';
 
 /*---------------------------------------------------------*/
```

Comment:
I added these I think but we could probably just get rid of them.

Reply:
lines 6 and 7? won't the code break?

Reply:
will do